xml/admin_monitoring_alerting.xml

```diff
@@ -366,36 +366,130 @@ placement:
     If you want to get notified about &prometheus; alerts via SNMP traps, then
     you can install the &prometheus; &alertmanager; SNMP trap receiver via
-    &cephadm;. To do so, you need to enable it in the Pillar under the
-    <option>monitoring:alertmanager_receiver_snmp:enabled</option> key. The
-    configuration of the receiver must be set under the
-    <option>monitoring:alertmanager_receiver_snmp:config</option> key.
+    &cephadm;. To do so, you need to create a service and placement
+    specification file with the following content:
    </para>
-   <example>
-    <title>SNMP Trap Configuration</title>
+   <note>
+     <para>
+       For more information on service and placement files, see
+       <xref linkend="cephadm-service-and-placement-specs"/>.
+     </para>
+   </note>
+<screen>
+service_type: container
+service_id: prometheus-webhook-snmp
+placement:
+    <replaceable>ADD_PLACEMENT_HERE</replaceable>
+image: registry.suse.com/ses/7/prometheus-webhook-snmp:latest
+args:
+    - "--publish 9099:9099"
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>ADD_HOST_GATEWAY_HERE</replaceable>"
+    - RUN_ARGS="--metrics"
+EOF
+</screen>
+     <para>
+      Use this service specification to get the service running using
+      its default settings.
+     </para>
+     <para>
+      You need to publish the port the &prometheus; receiver is listening on by
+      using the command line argument <literal>--publish <replaceable>HOST_PORT</replaceable>:<replaceable>CONTAINER_PORT</replaceable></literal>
+      when running the service, because the port is not exposed automatically by the
+      container. This can be done by adding the following lines to the specification:
+     </para>
 <screen>
-monitoring:
- alertmanager:
-   receiver:
-      snmp:
-        enabled: True
-        config:
-          host: localhost
-          port: 9099
-          snmp_host: snmp.foo-bar.com
-          snmp_community: private
-          metrics: True
+args:
+    - "--publish 9099:9099"
 </screen>
      <para>
-      Refer to the receiver manual at
-      <link xlink:href="https://github.com/SUSE/prometheus-webhook-snmp#global-configuration-file" />.
-      for more details about the configuration options.
+       Alternatively, connect the container to the host network by using
+       the command line argument <literal>--network=host</literal>.
      </para>
-    </example>
-  </sect1>-->
+<screen>
+args:
+    - "--network=host"
+</screen>
+     <para>
+      If the SNMP trap receiver is not installed on the same host as
+       the container, then you must also specify the FQDN of the SNMP
+       host. Use the container's network gateway to be able to receive SNMP
+       traps outside the container/host:
+     </para>
+<screen>
+envs:
+    - ARGS="--debug --snmp-host=<replaceable>CONTAINER_GATEWAY</replaceable>"
+</screen>
+     <sect2 xml:id="configure-prometheus-webhook-snmp">
+       <title>Configure the prometheus-webhook-snmp service</title>
```

Comment:
I suggest some kind of tag around `prometheus-webhook-snmp` such as literal or systemitem or something, followed by a capital S on Service, but I am not going to fight you for it. :-)